src/meshapi/admin/models/access_point.py

```diff
@@ -30,9 +32,10 @@ class Meta:
 
 
 @admin.register(AccessPoint)
-class AccessPointAdmin(ImportExportModelAdmin, ExportActionMixin):
+class AccessPointAdmin(RankedSearchMixin, ImportExportModelAdmin, ExportActionMixin):
     form = AccessPointAdminForm
-    search_fields = ["name__icontains", "notes__icontains"]
+    search_fields = ["name__icontains", "@notes"]
+    search_vector = SearchVector("name", weight="A") + SearchVector("notes", weight="D")
```

Comment:
Is there a difference between doing `A, D` and `A, B` if there's only two options?

Reply:
Yes, from the [Django docs on this feature](https://docs.djangoproject.com/en/5.1/ref/contrib/postgres/search/#postgresql-fts-weighting-queries):

> The weight should be one of the following letters: D, C, B, A. By default, these weights refer to the numbers 0.1, 0.2, 0.4, and 1.0, respectively

So by doing `A, D` instead of `A, B`, we weight the notes field about 4x lower. This gives a "fuzzy match" on the name field, which otherwise would get a low score an even better chance of winning against an exact match in the notes of a separate object. Effectively it weights the name *even higher* than the notes

I'm not 100% sure this is *always* the best. I can't find the example I worked off of now, but it seems to be the way people do this sort of "headine" vs "body contents" thing in other applications